src/pages/shop/Shop.tsx

```diff
@@ -1,15 +1,193 @@
+    setActive(!isActive);
+  };
+  const handleFilterToggle = () => {
+    setFilter(!isFilter);
+  };
+
+
   return (
-    <section className="Shop">
+    <section className="shop">
       <Banner
         title="Shop"
         pageName="Shop"
         coverPhoto="https://casona.familab.net/wp-content/uploads/2021/09/heading_bg_dark.jpg"
         sliderExist={true}
+        bannerHeight="440px"
       />
+      <div className="container">
+        <div className="shop-control">
+          <Flex gap="20"  justifyContent="space-between">
+            <button className="filtering" onClick={handleFilterToggle}>
+              <div className="filterDropdown">
+                <VscFilter className="filterIco" />
+                <span>FILTER</span>
+              </div>
+
+              <div id="filterList" className={isFilter ? 'active' : ''}>
+                <div  className='hideFilter'>
+                  <button className='closefilter'>
+                   <GrFormClose className='closeIcon' /><span>Hide Filters</span>
+                  </button>
+                  </div>
+                <Flex gap="20" alignItems="center" justifyContent="space-between">
+                  <div className="productCategories">
+                    <p>Product Categories</p>
+                    <div className="categoryList">
+                    <ul className='menuList'>
+                      <li>
+                        <Link to='/'>Accessories</Link>
+                      </li>
+                      <li>
+                        <Link to='/'>Backpacks</Link>
+                      </li>
+                      <li className='digital'>
+                        <Link to='/'>Digitals </Link>
+                        <button className='icon' ><MdOutlineKeyboardArrowDown/></button>
+                        <ul  id='subMenu'>
+                          <li>
+                            <Link to='/'>Headphones</Link>
+                          </li>
+                          <li>
+                            <Link to='/'>Speaker</Link>
+                          </li>
+                        </ul>
+                      </li>
+                      <li>
+                        <Link to='/'>Wallets</Link>
+                      </li>
+                      <li>
+                        <Link to='/'>Watches</Link>
+                      </li>
+                      <li>
+                        <Link to='/'>Water Bootles</Link>
+                      </li>
+                      <li>
+                        <Link to='/'></Link>
+                      </li>
+                    </ul>
+                    </div>
+                    
+                  </div>
+                  <div className="productTags">
+                    <p>Product Tags</p>
+                    <Stack direction='column'>
+                           <Wrap spacing={10}>
+                              <WrapItem>
+                                <Button className='btn'><span>Bosch</span></Button>
```

Comment:
I'm not sure that your components will be buttons. maybe it will be needed to change them to links